webrender/src/prim_store.rs

```diff
@@ -1754,7 +1754,9 @@ impl PrimitiveStore {
             let metadata = &mut self.cpu_metadata[prim_index.0];
             if metadata.local_rect.size.width <= 0.0 ||
                metadata.local_rect.size.height <= 0.0 {
-                warn!("invalid primitive rect {:?}", metadata.local_rect);
+                if metadata.prim_kind != PrimitiveKind::Picture {
```

Comment:
The local rect for a picture is updated in the block just above this, isn't it? Does it happen on all pictures?

Reply:
right now I can only say that it happens on a lot of pictures even with the other commit that removes empty rect at during dl building.

Reply:
Interesting! I *think* that is unexpected. I wonder if we end up getting a lot of stacking contexts pushed that are empty, resulting in zero-sized pictures or something like that. This is probably worth investigating a bit - it's possible that it's a sign of a deeper issue or something we can catch earlier for a slight performance win. Or it might just be a code that I'm not thinking of right now.